docs/faqs.md

```diff
@@ -27,7 +27,7 @@ We don’t know yet. We are open to doing so but we’d like to revisit this aft
 
 ### Does using multicloud mean you will use the lowest common denominator across clouds?
 
-Not necessarily. There are numerous best of breed cloud offerings that run on multiple clouds. For example, CockroachDB and ElasticSearch are world class implementations of platform software and run well on cloud providers. They compete with managed services offered by a cloud provider. We believe that by having a open control plane for they to integrate with, and providing a common API, CLI and UI for all of these services, that more of these offerings will exist and get first-class experience in the cloud.
+Not necessarily. There are numerous best of breed cloud offerings that run on multiple clouds. For example, CockroachDB and ElasticSearch are world class implementations of platform software and run well on cloud providers. They compete with managed services offered by a cloud provider. We believe that by having an open control plane for them to integrate with, and providing a common API, CLI and UI for all of these services, that more of these offerings will exist and get first-class experience in the cloud.
```

Comment:
is this just whitespace changes in this paragraph? my eyes aren't seeing a specific wording/grammar/punctuation change, but I may be missing it.

Reply:
Sorry about that, should have specified in PR body:

"We believe that by having ~~a~~ **an** open control plane for ~~they~~ **them** to integrate with, and providing a common API, CLI and UI for all of these services, that more of these offerings will exist and get first-class experience in the cloud."

Reply:
i see it now! lol.  i'm surprised that the diff isn't showing those intra-line changes here, but thanks for pointing them out.